### PR TITLE
docs - Add OpenJDK install information

### PR DESCRIPTION
**Pull request by mkiyama**
This will be backported to 6X_STABLE

Link to HTML format of draft changes
http://docs-msk-gpdb6-pljava-dev.cfapps.io/6-0/ref_guide/extensions/pl_java.html#topic_qx1_xcp_w3b

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/extensions/pl_java.xml [hunk 1]
```diff
@@ -167,6 +168,63 @@
       </body>
     </topic>
   </topic>
+  <topic id="topic_qx1_xcp_w3b">
+    <title>Installing Java</title>
+    <body>
+      <p>PL/Java requires a Java environment on each Greenplum Database host. Ensure that the same
```
[the remark this answers is not in the document]
Comment (mkiyama) [reply gpdb-doc/dita/ref_guide/extensions/pl_java.xml]: fixed

File: gpdb-doc/dita/ref_guide/extensions/pl_java.xml [hunk 2]
```diff
@@ -208,10 +261,9 @@
             <codeph>$MASTER_DATA_DIRECTORY</codeph> and <codeph>$GPHOME</codeph> variables are
           set.</p>
         <ol>
-          <li id="pv158485">Download the PL/Java extension package from the Greenplum Database page
-            on <xref href="https://network.pivotal.io/products/pivotal-gpdb" scope="external"
-              format="html" class="- topic/xref ">Pivotal Network</xref> then copy it to the master
-            host. </li>
+          <li id="pv158485">Download the PL/Java extension package from <xref
+              href="https://network.pivotal.io/products" scope="external" format="html"
```
[the remark this answers is not in the document]
Comment (mkiyama) [reply gpdb-doc/dita/ref_guide/extensions/pl_java.xml]: fixed

File: gpdb-doc/dita/ref_guide/extensions/pl_java.xml [hunk 3]
```diff
@@ -223,19 +275,18 @@
               <li>Set the <codeph>JAVA_HOME</codeph> variable to the directory where your Java
                 Runtime is installed. For example, for Oracle JRE this directory would be
                   <codeph>/usr/java/latest</codeph>. For OpenJDK, the directory is
-                  <codeph>/usr/lib/jvm</codeph>. This example changes the environment variable
-                to use
-                <codeph>/usr/lib/jvm</codeph>.<codeblock>JAVA_HOME=/usr/lib/jvm</codeblock></li>
+                  <codeph>/usr/lib/jvm</codeph>. This example changes the environment variable to
+                use <codeph>/usr/lib/jvm</codeph>.<codeblock>JAVA_HOME=/usr/lib/jvm</codeblock></li>
```
[the remark this answers is not in the document]
Comment (mkiyama) [reply gpdb-doc/dita/ref_guide/extensions/pl_java.xml]: added export

File: gpdb-doc/dita/ref_guide/extensions/pl_java.xml [hunk 4]
```diff
@@ -167,6 +168,63 @@
       </body>
     </topic>
   </topic>
+  <topic id="topic_qx1_xcp_w3b">
+    <title>Installing Java</title>
+    <body>
+      <p>PL/Java requires a Java environment on each Greenplum Database host. Ensure that the same
+        Java environment is at the same location on all hosts: masters and segments. The command
+          <codeph>java -version</codeph> displays the Java version.</p>
+      <p>You can install Java on the host systems. How to install Java depends on the host system
```
[the remark this answers is not in the document]
Comment (mkiyama) [reply gpdb-doc/dita/ref_guide/extensions/pl_java.xml]: updated

File: gpdb-doc/dita/ref_guide/extensions/pl_java.xml [hunk 5]
```diff
@@ -167,6 +168,63 @@
...
+                    <li>This example <codeph>tar</codeph> command installs the Java 11 JDK in
+                        <codeph>/usr/lib/jvm</codeph>.<codeblock>$ sudo tar xzf openjdk-11.0.2_linux-x64_bin.tar.gz --directory /usr/lib/jvm</codeblock></li>
+                    <li>Run these two commands to add Java 11 JDK to the
+                        <codeph>update-alternatives</codeph> command that maintains symbolic links
+                      that determines the default version of
+                        commands.<codeblock>$ sudo sh -c 'for bin in /usr/lib/jvm/jdk-11.0.2/bin/*; do update-alternatives --install /usr/bin/$(basename $bin) $(basename $bin) $bin 100; done'
+$ sudo sh -c 'for bin in /usr/lib/jvm/jdk-11.0.2/bin/*; do update-alternatives --set $(basename $bin) $bin; done'</codeblock><p>The
+                        second command returns some <codeph>failed to read link</codeph> errors that
+                        can be ignored.</p></li>
+                    <li>Run this command to change the default Java version to Java 11. Enter the
```
[the remark this answers is not in the document]
Comment (mkiyama) [reply gpdb-doc/dita/ref_guide/extensions/pl_java.xml]: revised Java install section

File: gpdb-doc/dita/ref_guide/extensions/pl_java.xml [hunk 6]
```diff
@@ -167,6 +168,55 @@
...
+              11.<codeblock>$ sudo apt update
+$ sudo apt install openjdk-&lt;<varname>version</varname>>-jdk</codeblock><p>For
+              OpenJDK 8 the <varname>version</varname> is <codeph>8</codeph>, for OpenJDK 11 the
+                <varname>version</varname> is <codeph>11</codeph>.</p></li>
+        </ul></p>
+      <p>After installing OpenJDK on a RHEL or CentOS system, run this
+          <codeph>update-alternatives</codeph> command to change the default Java. Enter the number
+        that represents the OpenJDK version to use as the
+        default.<codeblock>$ sudo update-alternatives --config java </codeblock></p>
+      <p>The <codeph>update-alternatives</codeph> command is not required on Ubuntu systems.</p>
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/extensions/pl_java.xml]: Not sure why we add this right after asking them to use the command.

Comment (mkiyama) [reply gpdb-doc/dita/ref_guide/extensions/pl_java.xml]: The command is needed on RHEL and CentOS, not Ubuntu.

**Pull request merged**